posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+![Figure 23 : editing the access level of a Container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_23.png)
+
+It is possible to change the accessibility level of a container even after it has been created by right-clicking on it, which can be very useful.
+
+Finally, enter the container to view its contents.
+
+![Figure 24 : content of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_24.png)
+
+This view lists all the blobs contained in a container. A container can contain three types of blobs namely _Block blobs_, _Append blobs_ and _Page blobs_.
+
+_Block blobs_ are designed to store text or binary files, and to load large files efficiently. _Append blobs_ are designed for append operations such as logging. Finally, _Page blobs_ are made up of pages of up to 512 bytes that can reach a total size of 8 TB. They are designed for frequent random read/write operations. In the context of our project, we only use _Block blobs_.
+
+Thus, the view of a container allows to visualize the blobs contained in it as well as the public URL at which they are accessible, but also to upload new ones, to delete them and even to create access tokens specific to these blobs.
+
+### Queues
+
+![Figure 25 : the queues of a storage](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_25.png)
+
+This view lists all the queues created in the storage. It allows you to create new queues and to delete them. Simulate the creation of a new queue by clicking on the '+' button next to the _Refresh_ button.
+
+![Figure 26 : creation of a new queue](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_26.png)
+
+This view is the interface for creating a new queue. Very simple, it just requires the name of the new queue. Save, it is created and usable.
+
+## Functions
+
+In the context of our project, we did not have the opportunity to use them, but you should know that Azure offers a service of _Function apps_ which are serverless applications that can be hosted on different runtimes such as Java, .Net, Node, Python or even Powershell.
+
+So you can create a _function app_ that acts as a trigger when a file is dropped in a container or when a message is dropped in a queue.
+
+# Conclusion
+
+Through this article, you have taken a first step into the Azure cloud offered by Microsoft by browsing the components useful to web application developers.
+
+In particular, you have studied the _web apps_ that host the code of your web application, the _DBs_ that store the data of your web app, the _storages_ that are useful for storing files and messages and you also know that the _functions_ are at your disposal to configure HTTP triggers on all types of events observed by your web app.
```

Comment:
In summary, you have studied the _web apps_ that host the **code** of your web application, the _DBs_ that store the **data** of your web app, the _storages_ that are useful for **storing** files and messages, and you are now aware that the _functions_ are at your disposal to configure HTTP triggers on all types of events observed by your web app.